StRoot/StPicoEvent/StPicoEvent.h

```diff
@@ -460,7 +508,31 @@ class StPicoEvent : public TObject {
   UShort_t mNTofT0;
   /// Vz estimated via VPD
   Float_t  mVzVpd;
-
+  /// Number of particles before T0 outlier rejection
+  UShort_t mNTofT0Can;
+  /// Event start time in ps
+  Int_t  mTStart; 
+  /// Earliest candidate particle in T0 calculation
+  Int_t  mTCanFirst;
+  /// Last candidate particle in T0 calculation
+  Int_t  mTCanLast;
+  /// Total number of in-time VPDE hits
+  UChar_t  mNVpdEGoodHits;
+  /// Total number of in-time VPDW hits
+  UChar_t  mNVpdWGoodHits;
+  /// Earliest VPDE hit compared to start time
+  Int_t  mEarliestVpdEHit;
+  /// Earliest VPDW hit compared to start time
+  Int_t  mEarliestVpdWHit;
+  /// Closest VPDE hit to start time
+  Int_t  mClosestVpdEHit;
+  /// Closest VPDW hit to start time
+  Int_t  mClosestVpdWHit;
+  /// Furthest VPDE hit compared to start time
+  Int_t  mLatestVpdEHit;
+  /// Furthest VPDW hit compared to start time
+  Int_t  mLatestVpdWHit;
+ 
```

Comment:
Should ClassDef version increase when new member are added?

Reply:
Yeah! I am pretty sure that it should be if the .h file is edited. I was not sure if it is something that I should do on my end, or if it is something the code owners do.

Reply:
I would be more than happy to do it. Just let me know your preference.

Reply:
Hi Bassam, please do it.

Reply:
No problem! Should I increment only the first ClassDef in the following from StPicoEvent.h?

![Screen Shot 2021-06-15 at 9 12 52 AM](https://user-images.githubusercontent.com/85306311/122068288-e506bd00-cdb9-11eb-97f0-9c1cf5182cb2.png)

Reply:
Please do it for both.